internal/printer/utilities.go

```diff
@@ -103,8 +103,18 @@ func escapeStringWorker(s string, quoteChar QuoteChar, flags getLiteralTextFlags
 				escape = true
 			}
 		default:
-			if ch < '\u001f' || flags&getLiteralTextFlagsNeverAsciiEscape == 0 && ch > '\u007f' {
-				escape = true
+			// For template strings, exclude \u000a (LF/newline) from the character range that gets escaped
```

Comment:
This doesn’t look right. Please think about two questions:

1. Are _any_ code changes necessary to make `\n` work correctly in template strings?
2. Is `\u001f` working correctly?